superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -163,6 +198,15 @@ const RightMenu = ({
                     className="data-menu"
                     title={menuIconAndLabel(menu)}
                   >
+                    <Menu.Item key={GlobalMenuDataOptions.DB_CONNECTION}>
+                      Connect Database
+                    </Menu.Item>
+                    {SHOW_GLOBAL_GSHEETS && (
+                      <Menu.Item key={GlobalMenuDataOptions.GOOGLE_SHEETS}>
+                        Connect Google Sheet
+                      </Menu.Item>
+                    )}
+                    <Menu.Divider />
```

Comment:
I'd suggest since we have the data object above that we're looping over, to try to move this into that data structure so that we can keep this as modular as possible.